server/app/api/v1/endpoints/calls.py

```
@@ -1257,6 +1258,214 @@ async def bulk_upload_calls(
+        logger.info(f"Call details retrieved: {call_details}")
+        
+        # Check if recording URL exists
+        recording_url = call_details.get("recording_url")
+        logger.info(f"Recording URL from database: {recording_url}")
+        
+        if not recording_url:
+            logger.error(f"No recording URL found for call {call_id}")
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="No recording available for this call"
+            )
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)